### DISCUSSION TITLE
---

### DISCUSSION THREAD

**Discussion by Freakwill** [Homebrew/discussions#4415] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.13-13-g688b732
ORIGIN: https://github.com/Homebrew/brew
HEAD: 
Last commit: 3 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.w9vnsstky1/org.xquartz:0
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_INSTALL_FROM_API: set
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 12.0.0 build 1200
Git: 2.40.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.7-x86_64
CLT: 12.0.0.32.29
Xcode: N/A
```

### Output of `brew doctor`

```shell
Warning: You are using macOS 10.15.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.
```

### Description of issue

Following is the error when running `brew upgrade python@3.10`. How do I fix it?
> Error: python@3.10: Failed to download resource "python@3.10--patch"
> Failure while executing; `/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.0.13-13-g688b732\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15.7\)\ curl/7.64.1 --header Accept-Language:\ en --retry 3 --fail --location --silent --head https://raw.githubusercontent.com/Homebrew/formula-patches/a1618a5005d0b01d63b720321806820a03432f1a/python/3.10-sysconfig.diff` exited with 60. Here's the output:
> curl: (60) SSL: no alternative certificate subject name matches target host name 'raw.githubusercontent.com'
> More details here: https://curl.haxx.se/docs/sslcerts.html
> 
> curl failed to verify the legitimacy of the server and therefore could not
> establish a secure connection to it. To learn more about this situation and
> how to fix it, please visit the web page mentioned above.

**Comment by SMillerDev** [comment]
> curl: (60) SSL: no alternative certificate subject name matches target host name 'raw.githubusercontent.com'
> More details here: https://curl.haxx.se/docs/sslcerts.html

The information is in the error message already.
